lib/krane/kubernetes_resource/cloudsql.rb

```diff
@@ -2,6 +2,7 @@
 module Krane
   class Cloudsql < KubernetesResource
     TIMEOUT = 10.minutes
+    SYNC_DEPENDENCIES = %w(Deployment Service)
```

Comment:
How do you resolve the dependencies of those types of resource?

Reply:
We go through the sync loop for that resource and have a look at what ones it's fetching. In the case of cloudsql we see [Service and Deployment](https://github.com/Shopify/kubernetes-deploy/blob/master/lib/krane/kubernetes_resource/cloudsql.rb#L8-L9) so we prewarm those! 👍 
Does that answer your question?

Reply:
Yes, that answers the question 😄 , I thought you looked for some k8s document

Reply:
The PR looks great, :shipit: